client/my-sites/comments/comment/comment-edit.jsx

```diff
@@ -41,20 +46,46 @@ export class CommentEdit extends Component {
 		authorDisplayName: '',
 		authorUrl: '',
 		commentContent: '',
+		commentDate: '',
+		isDatePopoverVisible: false,
+		tmpCommentDate: '',
 	};
 
 	componentWillMount() {
-		const { authorDisplayName, authorUrl, commentContent } = this.props;
-		this.setState( { authorDisplayName, authorUrl, commentContent } );
+		const { authorDisplayName, authorUrl, commentContent, commentDate } = this.props;
+		this.setState( { authorDisplayName, authorUrl, commentContent, commentDate } );
 	}
 
+	storeDatePopoverButtonRef = button => ( this.datePopoverButtonRef = button );
```

Comment:
check out the `createRef()` API which is now preferred

https://reactjs.org/docs/react-api.html#reactcreateref

Reply:
Ooh! I didn't know about that!
Will they ever stop changing the refs api? 😄